src/FINDOLOGIC/Api/Responses/Xml21/Properties/Filter/ImageFilter.php

```diff
@@ -2,6 +2,6 @@
 
 namespace FINDOLOGIC\Api\Responses\Xml21\Properties\Filter;
 
-class VendorImageFilter extends Filter
+class ImageFilter extends Filter
```

Comment:
Please rename this back to `VendorImageFilter`, because the filter itself can only be `image` for a `vendor`.